python/flare/idb2pat.py

```diff
@@ -234,13 +234,16 @@ def make_func_sig(config, func):
         if name is None or name == "":
             continue
 
+        addr = 0
+        ref_format = ""
+        
```

Comment:
can this be removed?

not a blocker, just wondering

Reply:
I think it does not matter, var scope is the whole function in python. I just wanted to have a visible remark they are proper reset during each turn. I found this while tracking down another issue which turned out to be a bug in sigmake in the end.